### ISSUE TITLE
Kruskal's MST

### ISSUE THREAD

**Issue by SamareshMaiti16** [SamareshMaiti16/PythonDSA#18]
Write the code to perform Kruskal's MST in python.
Guidelines:

- Write the codes in python only.
- Refrain from any unfair means or plagiarism.
- The code should display authenticity.
- Use proper formatting and comments.

*Note: MST stands for Minimum Spanning Tree.

**Comment by Sohom2002** [comment]
Hi @SamareshMaiti16.. would like to contribute in this one... The Kruskal's Algorithm is a very fascinating one, unlike the Prims Algorithm. This ones actually easy to interpret and understand for the beginners.

**Comment by SamareshMaiti16** [comment]
Yes @Sohom2002 kruskal's algo is an interesting one indeed. Go ahead get started. All the best.
